core/dbt/task/show.py

```diff
@@ -0,0 +1,81 @@
+import io
+import threading
+
+from dbt.contracts.results import RunResult, RunStatus
+from dbt.events.functions import fire_event
+from dbt.events.types import ShowNode
+from dbt.exceptions import DbtRuntimeError
+from dbt.task.compile import CompileTask, CompileRunner
+
+
+class ShowRunner(CompileRunner):
+    def __init__(self, config, adapter, node, node_index, num_nodes):
+        super().__init__(config, adapter, node, node_index, num_nodes)
+        self.run_ephemeral_models = True
+
+    def execute(self, compiled_node, manifest):
+        adapter_response, execute_result = self.adapter.execute(
+            compiled_node.compiled_code, fetch=True
+        )
+
+        return RunResult(
+            node=compiled_node,
+            status=RunStatus.Success,
+            timing=[],
+            thread_id=threading.current_thread().name,
+            execution_time=0,
```

Comment:
This seems incorrect? Is this just for expediency, because the way we do `timing` + `execution_time` measurement elsewhere is very complicated?